howler.go

```diff
@@ -0,0 +1,118 @@
+package main  
+
+import (
+	// "fmt";
+    "log"
+    "github.com/fsnotify/fsnotify"
+    "github.com/bgveenstra/slacker"
+    "fmt"
+    "os"
+    "path/filepath"
+)
+// https://github.com/fsnotify/fsnotify/blob/master/example_test.go
+func main() {
+	log.Println("begin main")
+	watchDirArg := os.Args[1]
+
+	WatchDirForever(watchDirArg)
+	log.Println("end main")
+}
+
+
+func WatchDirForever(dir string){
+
+	eventWatchTypes := make(map[string] bool)
+	// watch types should come from flag or config - some arg
+	eventWatchTypes["CREATE"] = true
+
+
+	watchDir, err := filepath.Abs(dir)
+	if err != nil {
+		log.Fatal(err)
+	}	
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Fatal(err)
+	} 
+
+	// defer statement pushes a function call onto a list
+	// list of saved calls is executed after the surrounding function returns
+	// good for clean-up actions
+	// here, closing the watcher after this WatchDirForever function is finished
+	// Close removes all watches and closes the events channel. 
+	defer watcher.Close()
+
+
+
+	log.Println("before")
+	// invoking anonymous function in a goroutine
+	// executes alongside the rest of this function's code
+	go func() {
+		log.Println("goroutine!")
+		for { // while true
+			select {
+				// read from watcher.Events (channel of Events)
+				case event := <-watcher.Events:
+					// if there is an event to be had, log it
+					log.Println("event name:", event.Name)
+					log.Println("event op:", event.Op)
+					if eventWatchTypes[event.Op.String()] {
+						slacker.PostSlackMessage(fmt.Sprintf("Holwer Update \n%s %s", event.Op, event.Name))
```

Comment:
Maybe do some better formatting on the name here.  For example, for my use case, I don't need "Howler update" appended to each message.  Instead, maybe try to make the message more human readable, like:
```go
var MessageFmt := "New file found: %s"
slacker.PostMessage(fmt.Sprintf(MessageFmt, event.Name)
```

Since we know it is a create due to your event filtering.